potku.py

```diff
@@ -1485,6 +1486,28 @@ def __open_manual(self):
                 self, "Not found",
                 "There is no manual to be found!",
                 QtWidgets.QMessageBox.Ok, QtWidgets.QMessageBox.Ok)
+                
+                
+    def __open_data_help(self):
+        """Open data help file.
+        """
+        # TODO duplicate code with __open_manual
+        # TODO changed the file path to point to the manual, I guess this needs
+        #      to be updated in the .spec file too?
+        manual_filename = Path("documentation", "Potku_data_explained.pdf")
```

Comment:
This path may not work in Mac application. You could change it to
```python
file_name = gf.get_root_dir() / "documentation" / "Potku_data_explained.pdf"
```
(add `import modules.general_functions as gf` if the module is not already imported)